src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs

```diff
@@ -104,10 +104,10 @@ private void plotDataSet(object sender, DataSetSelectedEventArgs e)
          }
          catch (InvalidArgumentException invalidException)
          {
-            _view.DisableConfirmationView();
             var errors = new ParseErrors();
             errors.Add(_dataSource.DataSetAt(e.Index), new NonMonotonicalTimeParseErrorDescription(Error.ErrorWhenPlottingDataRepository(e.Index, invalidException.Message)));
```

Comment:
I know it does not come from this commit...but WHY is there this explicit `NonMonotonicalTimeParseErrorDescription` coming out of nowhere??